internal/dag/dag.go

```diff
@@ -81,6 +81,9 @@ type Route struct {
 
 	// Indicates that during forwarding, the matched prefix (or path) should be swapped with this value
 	PrefixRewrite string
+
+	// Defines ratelimiting configuration if specified
+	RateLimitConfiguration map[string]string
```

Comment:
Is there a stronger type that can be used here?

Reply:
There is, but was trying to not make it specific to the Lyft version of RateLimiting. Folks could implement their own service. Let me look and see if I can make it more specific but still generic.